client/jetpack-cloud/sections/manage/pricing/license-products-list/featured-license-multi-item-card.tsx

```diff
@@ -0,0 +1,114 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button } from '@automattic/components';
+import 'calypso/my-sites/plans/jetpack-plans/product-store/featured-item-card/style.scss';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useState } from 'react';
+import MultipleChoiceQuestion from 'calypso/components/multiple-choice-question';
+import { useProductDescription } from 'calypso/jetpack-cloud/sections/partner-portal/hooks';
+import getProductShortTitle from 'calypso/jetpack-cloud/sections/partner-portal/lib/get-product-short-title';
+import getProductVariantShortTitle from 'calypso/jetpack-cloud/sections/partner-portal/lib/get-product-variant-short-title';
+import { HeroImageAPIFamily } from 'calypso/my-sites/plans/jetpack-plans/product-store/hero-image';
+import { useDispatch } from 'calypso/state';
+import { APIProductFamilyProduct } from 'calypso/state/partner-portal/types';
+import { ItemPrice } from './item-price';
+import './style.scss';
+
+type FeaturedLicenseMultiItemCardProps = {
+	items: APIProductFamilyProduct[];
+	bundleSize?: number;
+	ctaAsPrimary?: boolean;
+	ctaHref?: string;
+	moreInfoLink?: React.ReactNode;
+	isCondensedVersion?: boolean;
+	isCtaDisabled?: boolean;
+	isCtaExternal?: boolean;
+	onClickCta?: VoidFunction;
+};
+
+export const FeaturedLicenseMultiItemCard = ( {
+	items,
+	bundleSize,
+	ctaAsPrimary,
+	ctaHref,
+	moreInfoLink,
+	isCtaDisabled,
+	isCtaExternal,
+	onClickCta,
+}: FeaturedLicenseMultiItemCardProps ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const [ item, setItem ] = useState( items[ 0 ] );
+
+	let title = item.name;
+	if ( title.startsWith( 'Jetpack Security' ) ) {
+		title.replace( 'Jetpack Security', 'Security' );
+	}
+	if ( title.startsWith( 'Jetpack VaultPress' ) ) {
+		title.replace( 'Jetpack VaultPress', 'VaultPress Backup' );
+	} else {
+		title = getProductShortTitle( item, true );
+	}
```

Comment:
Unless I'm missing something, I think this whole section can be simplified:

```suggestion
	let title = getProductShortTitle( item, true );
```

Reply:
Looks like it, yes. Seemed to need it earlier but I'm wondering if when testing titles initially the cards weren't rendering properly as I started experiencing that later (I had to manually add a bundleSize of eg 5 to the most-popular-plans file and the all-license-items file as the result using the bundleSize in the main index.tsx file was a mesh between multi and single product cards 😕 ).I imagined that when linking with license numbers via the selector later on, that we'd be making sure the correct state is being passed down then, but  I'll see what I uncover in the meantime.
Fixed this in this commit: e9dd3afd5dbbda87b34d96ab04efea54e203f064